core/src/main/java/fr/sncf/osrd/standalone_sim/StandaloneSim.java

```diff
@@ -61,8 +61,8 @@ public static StandaloneSimResult run(
                 var rollingStock = trainSchedule.rollingStock;
 
                 // MRSP & SpeedLimits
-                var mrsp = computeMRSP(trainPath, rollingStock, true, trainSchedule.tag);
-                var speedLimits = computeMRSP(trainPath, rollingStock, false, trainSchedule.tag);
+                var mrsp = computeMRSP(trainPath, rollingStock, true, trainSchedule.tag, null);
```

Comment:
This is java code, there's no default value.

This is also why the default value is `null` instead of an empty range map. I didn't feel like adding `distanceRangeMapOf(new ArrayList())` to every call.

Reply:
Yaaaay jaaavvvvaaa :D